DataProcessing/datax-host/src/main/scala/datax/input/BlobPointerInput.scala

```diff
@@ -48,7 +48,7 @@ object BlobPointerInput {
     }
   }
 
-  private def extractTimeFromBlobPath(blobPath: String, fileTimeRegex: Regex, fileTimeFormat: String): Timestamp = {
+  def extractTimeFromBlobPath(blobPath: String, fileTimeRegex: Regex, fileTimeFormat: String): Timestamp = {
```

Comment:
Why the access modifier has been changed? Does not look like this is being used outside the class